src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -243,6 +251,29 @@ def _agentpool_format_group(item):
     ])
 
 
+def _connected_acr_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('STATUS', _get_value(item, 'status')),
+        ('PARENT', _get_value(item, 'parent')),
+        ('LOGIN NAME', _get_value(item, 'loginName')),
+        ('AUTO UPDATE', _get_value(item, 'autoUpdate')),
+        ('LAST UPDATED', _get_value(item, 'lastUpdated')),
+        ('NEXT UPDATE', _get_value(item, 'nextUpdate')),
```

Comment:
What property does this correspond to? `NextSyncTime` is an internal only property

Reply:
Not sure yet. I'm following the cli-experience.md
https://msazure.visualstudio.com/AzureContainerRegistry/_git/functional-specs?path=%2FRegistry%2FConnected-ACR%2Fcli-experience.md&_a=preview